config/seata-config-nacos/src/test/java/io/seata/config/extend/TestConfigFromExtendSPI.java

```diff
@@ -56,33 +57,52 @@ public static void setup() throws NacosException {
         ConfigurationFactory.reload();
         ConfigurationCache.clear();
         FILE_CONFIG = ConfigFactory.load("registry-test.conf");
-        configService = NacosFactory.createConfigService(NacosConfiguration.getConfigProperties());
+        // 创建Mock对象
+        configService = Mockito.mock(ConfigService.class);
     }
 
     @Test
-    @EnabledOnOs(OS.LINUX)
```

Comment:
I recommend avoiding mock-based testing here. Please make it possible to run this as an integration test in an appropriate environment, following the example of the RedisQueryConsolTest class.